internals/Cargo.toml

```diff
@@ -18,9 +18,15 @@ default = []
 std = ["alloc"]
 alloc = []
 
+test-serde = ["serde/derive", "serde_json", "bincode"]
+
 [dependencies]
 serde = { version = "1.0.103", default-features = false, optional = true }
 
+# Don't enable these directly, use `serde` feature instead.
```

Comment:
This should say `test-serde` instead.

Reply:
bah I did perl one liner during dev to `s/test-serde/serde` and it was an epic fail. Thanks for catching me.